apps/rush-lib/src/cli/actions/ChangeAction.ts

```diff
@@ -100,32 +108,192 @@ export class ChangeAction extends BaseRushAction {
         'determine which projects were changed. If this parameter is not specified, the checked out branch ' +
         'is compared against the "master" branch.'
     });
+
+    this._overwriteFlagParemter = this.defineFlagParameter({
+      parameterLongName: '--overwrite',
+      description: `If a changefile already exists, overwrite without prompting ` +
+        `(or erroring in ${BULK_LONG_NAME} mode).`
+    });
+
+    this._changeEmailParameter = this.defineStringParameter({
+      parameterLongName: '--email',
+      argumentName: 'EMAIL',
+      description: 'The email address to use in changefiles. If this parameter is not provided, the email address ' +
+        'will be detected or prompted for in intractive mode.'
+    });
+
+    this._bulkChangeParameter = this.defineFlagParameter({
+      parameterLongName: BULK_LONG_NAME,
+      description: 'If this flag is specified, apply the same change message and bump type to all changed projects. ' +
+        `The ${BULK_MESSAGE_LONG_NAME} and the ${BULK_BUMP_TYPE_LONG_NAME} parameters must be specified if the ` +
+        `${BULK_LONG_NAME} parameter is specified`
+    });
+
+    this._bulkChangeMessageParameter = this.defineStringParameter({
+      parameterLongName: BULK_MESSAGE_LONG_NAME,
+      argumentName: 'MESSAGE',
+      description: `The message to apply to all changed projects if the ${BULK_LONG_NAME} flag is provided.`
+    });
+
+    this._bulkChangeBumpTypeParameter = this.defineChoiceParameter({
+      parameterLongName: BULK_BUMP_TYPE_LONG_NAME,
+      alternatives: [...Object.keys(this._getBumpOptions()), ChangeType[ChangeType.none]],
```

Comment:
`ChangeType[ChangeType.none]` is comparing against a source code identifier, and making it part of the command-line contract.  This seems awkward and brittle.

I see that this practice predates your PR.  But maybe we should consider changing the `ChangeType` values to be strings instead of enums?  Or adding some helper functions for parsing/printing the enum values?  (That probably belongs in a separate refactoring PR.)

Reply:
Yeah we should probably refactor that. That bothered me a little bit.

Reply:
To be addressed in a separate refactoring PR.